_posts/2017-08-01-visualization-framework-suite.md

```diff
@@ -0,0 +1,105 @@
+
+Each framework in the suite now has a drop down in its header bar to allow easy navigation to the other frameworks.
+
+In addition, each framework has been assigned a color scheme, giving the user a visual cue to where he or she is in the suite (i.e. which framework's docs are currently being shown). This should be helpful now that the different frameworks follow the same documentation style.
+
+
+
+## New Releases
+
+In concert with the unification of the framework documentation websites, we are also releasing new versions of some of the frameworks in the suite.
+
+
+### react-map-gl v3
+
+A major under-the-hood upgrade, this release also makes more Mapbox GL features available to React users, significantly strengthens the React encapsulation of Mapbox GL, simplifies installation, and adds support for some advanced use cases.
+
+Release highlights:
+* The documentation and website are completely rewritten (and made part of the suite, of course)
+* Multi-touch (pinch to zoom and rotate etc) is now supported (e.g. for mobile devices)
+* New React components are available (Markers, Popups etc), matching the native API.
+* A new architecture supports some advanced use cases, like automatically hiding the map when tilting beyond the mapbox 60 degree limit, something we use in "hybrid" applications that layer 3D visualizations on top of mapbox (using deck.gl, of course).
+
+
+### luma.gl v4
+
+v4 brings complete support for WebGL2 (which represents a major upgrade of the WebGL API). But there are a number of other significant improvements that round off this major release.
+
+Release highlights:
+* Full WebGL2 Support
+* WebGL Capability Management
+* WebGL State Management
+* GLSL Module System
+* Debug and Profiling Support
+* Library Size Optimizations
+
+That said, many users will probably appreciate the dramatically improved luma.gl documentation more than any single feature.
+
+
+### deck.gl v4.1
+
+deck.gl v4.1 is now based on luma.gl v4. The blog post [introducing deck.gl v4](http://uber.github.io/deck.gl/blog/2017/introducing-deckgl-v4) mentioned our plans to introduce more GPU based computing into future versions deck.gl (e.g. GPU based data processing and aggregation), and the inclusion of luma.gl v4 is a major enabler for that effort.
+
+The new release also  adds a couple of previously "teased" WebGL2 example layers, the [Wind Map](http://uber.github.io/deck.gl/blog/2017/wind-map) layers. These layers are now offered as part of a reusable, stand-alone example.
+
+
+### react-vis
+
+react-vis has already quietly been aligning with the documentation style introduced by deck.gl v3 and v4 and is also one of Uber's most popular repositories, so including it in the suite was an obvious choice. No new specific release was made, we've just added the link menu to the react-vis website.
```

Comment:
### react-vis 1.7

The priority of version 1.7 is documentation. Every documentation page has been written from scratch, entirely rewritten or reviewed since version 1.6.  
Since version 1.5, we've been publishing react-vis as a bundle, making it easy to use on sandbox environments such as [codepen](https://codepen.io/ubervisualization/pen/BZOeZB) or [codesandbox](https://codesandbox.io/s/QWRnG5zg7) for quick experimentation.